nginx.conf

```
@@ -0,0 +1,43 @@
+
+error_log  logs/error.log  debug;
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+    #gzip  on;
+
+    # DataDome configuration
+    lua_ssl_verify_depth 3;
+    lua_ssl_trusted_certificate /etc/ssl/certs/ca-certificates.crt;
+
+    resolver 8.8.8.8;
+    init_by_lua_block {
+        require("datadome_openresty").setDatadomeConf( { datadome_server_side_key = "server_side_key", } )
```

```suggestion
        require("datadome_openresty").setDatadomeConf( { datadome_server_side_key = "datadome_server_side_key", } )
```
Can this be passed as a environment variable ?

---

commited the change.
The idea was to simplify the Dockerfile and have something closer to what the clients do (change this file manually in PROD).